README.md

```diff
@@ -71,21 +71,20 @@ This code would make most sense as a...
 - ...helper method.
 
 ```text
-Your answer...
+I would use the controller method because the controller looks at requests and works out what data to show and what views to render....
```

Comment:
I think the best option here is helper method - because it is concerned with presentation